fe/fe-core/src/main/java/org/apache/doris/backup/HdfsStorage.java

```diff
@@ -17,17 +17,484 @@
 
 package org.apache.doris.backup;
 
+import org.apache.commons.collections.map.CaseInsensitiveMap;
+import org.apache.doris.analysis.StorageBackend;
+import org.apache.doris.catalog.AuthType;
 import org.apache.doris.common.UserException;
+import org.apache.doris.common.util.BrokerUtil;
+import org.apache.doris.common.util.URI;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.file.FileVisitOption;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.List;
 import java.util.Map;
```

Comment:
import order should be 
···
org.apache.doris
<blank line>
third party package
<blank line>
standard java package
<blank line>
···